<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow</class>
 <widget class="QMainWindow" name="mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SmartShop</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../../OneDrive/Bilder/smartshoplogo.png</normaloff>../../../../../OneDrive/Bilder/smartshoplogo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="get_ingredients_button">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-style: solid;
	border-width: 2px;
	border-color: #9999aa;
	border-radius: 10px;
	color: white;
	background-color: #3474eb
}

QPushButton:enabled {
	background-color: #3474eb
	color: white;
}

QPushButton:pressed {
	background-color: #0d2f72;
	color: #fffffe;
}

QPushButton:hover:!pressed {
	background-color: E1F4FF;
	color: 0c2f70;
}

QPushButton:disabled {
	background-color: #aaaaaa;
	color: #ffffff;
}</string>
    </property>
    <property name="text">
     <string>Get ingredients</string>
    </property>
   </widget>
   <widget class="QComboBox" name="chosen_recipe">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Tacos</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hamburger</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Recept</string>
    </property>
   </widget>
   <widget class="QLabel" name="recipe_step">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>190</y>
      <width>131</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>460</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
